### DISCUSSION TITLE
Meeting tomorrow

### DISCUSSION THREAD

**Discussion by DrBubbles42** [Jshel/CEN5035Project#59] [General]
Let's meet tomorrow and discuss what we need to complete for sprint 3.

**Comment by DrBubbles42** [comment]
Lets plan on meeting sometime after class or in the evening around 8 if that works for everyone

**Reply by Jshel** [reply]
This works for me!

**Comment by DrBubbles42** [comment]
## Meeting Summery
### Frontend
#### Todo
-Add an upload file page.
-Make spot to view PDF.
#### Completed
-Admin portal to view contracts and messages

### Backend
#### Todo
-Make endpoints to send and receive messages.
#### Completed
-Contract upload and download
-Redirection and sessions

### Everyone
-Merge to main
